apps/web/next.config.js

```diff
@@ -262,6 +262,30 @@ const nextConfig = {
 
     return redirects;
   },
+  async headers() {
+    const ContentSecurityPolicy = `
+      default-src 'self';
+      script-src 'self';
+      child-src app.cal.com;
+      style-src 'self' app.cal.com;
+      font-src 'self';  
+    `;
+    return [
+      {
+        source: "/:path*",
+        headers: [
+          {
+            key: "X-Frame-Options",
+            value: "SAMEORIGIN",
```

Comment:
It certainly needs to be restricted to outside Booking Pages. It is applicable to all the pages of app right now.

Reply:
yea we need to exclude our embeddable pages

Reply:
Discussed internally, we need to do it for login page only. Would move this to middleware which is aware of routes